### PR TITLE
fix(ci): update commands

### PR DESCRIPTION
**Pull request by seaerchin**
## Problem
CI tests were failing as `docker compose up` is timing out despite working fine on local

## Solution
don't use docker compose, just pass a command directly

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -86,8 +86,8 @@ jobs:
           path: ~/.npm
           key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
       - run: npm ci
-      - run: docker compose -f docker-compose.test.yml up
-      - run: . .env.test && npx jest --runInBand
+      - run: npm run test:services
+      - run: npm run test
```
Comment (harishv7) [inline .github/workflows/ci.yml:90]: shouldn;t it be just npm run test, why do we have test:services?

Comment (seaerchin) [reply .github/workflows/ci.yml:90]: we set up a database for integration tests

**Comment by QiluXie** [comment]
Observed the CI failing yesterday afternoon, wondering how it happened all of a sudden. 

@harishv7 Does this look good to you? 
If so, can we merge it so the CI can run on other PRs in the repo? Now the CI is failing on every PR

**Review by QiluXie — APPROVED** [review]
CI passed all green. lgtm

**Comment by seaerchin** [comment]
### Merge activity

* **Dec 20, 3:51 AM**: @@seaerchin merged this pull request with [Graphite](https://app.graphite.dev/github/pr/isomerpages/isomercms-backend/1069).

**Pull request merged**